FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -420,6 +420,8 @@ - (void)fetchLatestConfig:(NSInteger)remainingAttempts targetVersion:(NSInteger)
         [strongSelf autoFetch:remainingAttempts - 1 targetVersion:targetVersion];
       }
     } else {
+      bool excludeEtagHeaderForRealtime =
```

Comment:
maybe also worth adding a comment here explaining when template version would be 0

Reply:
Done